src/components/RowsPerPageToggleButton.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
is this used anymore?

Reply:
Nope, I'll delete it, sry forgot to do that :(.